javascript-constructors/example-constructor.js

```diff
@@ -0,0 +1,7 @@
+function ExampleConstructor() {}
+console.log('log the value of prototype property of the ExampleConstructor function to the console: ', Object.getPrototypeOf(ExampleConstructor));
+console.log('lgo the value of the prototype property of variable to console: ', typeof Object.getPrototypeOf(ExampleConstructor));
```

Comment:
For these chunks, you should do `ExampleConstructor.prototype` rather than `Object.getPrototypeOf(ExampleConstructor))`